### ISSUE TITLE
Tooltip component needs aria-label even when not using Carbon icon

### ISSUE THREAD

**Issue by rstrangfeld** [carbon-design-system/carbon#2508]
The Tooltip component lets you use either (option 1) a Carbon icon, or (option 2) something else that you supply, as the displayed trigger element. Internally the Tooltip component wraps that element in a div that handles clicks by displaying the tooltip.

If you choose option 1, a carbon icon, the wrapper div gets whatever aria-label the caller specified as "iconDescription" for the Tooltip component. This behavior was added to the Tooltip to fix https://github.com/IBM/carbon-components-react/issues/1697.

However, if you choose option 2 and supply your own trigger element, then the wrapper div doesn't get any aria-label. It fails the "All interactive elements must have an interactive name" accessibility test, just like option 1 did before 1697.

The fix for 1697 was good, but it only fixed the issue for one of the two cases. Option 2 needs a similar fix.

Note that you can't just put the aria-label on the trigger element you supply... it's the wrapper div created by Tooltip that is the interactive element.

The fix for 1697 was a one-line change. That will probably be the case here too.

**Comment by joshblack** [comment]
Hey all! Going to close this out as I believe it's addressed. Feel free to comment if that's not the case and I'd be happy to re-open. Steps to reproduce would also help out a ton to help us track down what we need to, either in a codesandbox or equivalent environment :pray:
